Incremental.md

```diff
@@ -2,26 +2,29 @@
 
 We would like to avoid redrawing all row(s) when an event occurs.
 Currently, we redraw all row(s) except when:
-  * a character is inserted at the end of input (and there is no hint and no `highlight_char`),
-  * only the cursor is moved (input is not touched and no `highlight_char`).
+
+- a character is inserted at the end of input (and there is no hint and no `highlight_char`),
+- only the cursor is moved (input is not touched and no `highlight_char`).
 
 Ideally, we would like to redraw only impacted row(s) / cell(s).
 
 ### Observable values
 
 Currently, we assume that highlighting does not impact layout / rendered text size.
 So only the following observables impact layout:
-  * prompt (interactive search, [input mode indicator](https://github.com/kkawakam/rustyline/pull/369)),
-  * [input mode](https://github.com/kkawakam/rustyline/pull/369),
-  * line(s) buffer,
-  * cursor position,
-  * hint / input validation message,
-  * screen size (line wrapping),
-  * [prompt continuation](https://github.com/kkawakam/rustyline/pull/372)s,
-  * row/wrap count.
+
+- prompt (interactive search, [input mode indicator](https://github.com/kkawakam/rustyline/pull/369)),
+- [input mode](https://github.com/kkawakam/rustyline/pull/369),
+- line(s) buffer,
+- cursor position,
+- hint / input validation message,
+- screen size (line wrapping),
+- [prompt continuation](https://github.com/kkawakam/rustyline/pull/372)s,
+- row/wrap count.
 
 Some other values may impact layout but they are/should be constant:
-  * tab stop,
+
```

Comment:
Not sure the extra line is prettier...

Reply:
It's related to [Incremental.md:26: MD032 Lists should be surrounded by blank lines](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md032---lists-should-be-surrounded-by-blank-lines).